### DISCUSSION TITLE
separate log files

### DISCUSSION THREAD

**Discussion by Hardcore-fs** [Chia-Network/chia-blockchain#8691] [Ideas]
can we PLEASE have separate log files so that 
"spammy" error messages" go into a SEPARATE log file , rather than being continually spammed at length into the general log file that tracks  transactions/proofs  etc

It's getting a it much to have to plow thru tens of thousand of error messages related to poor coding of trapping disconnects.

**Comment by FuzeGuy** [comment]
You can do this yourself for all intents and purposes. Edit the config.yaml to add the specifics of how you want a separate file output and include that in each .yaml section, as required, i.e., such as for the harvester section below. You'll have separate output file for each done this way. Modify the parameters to suit in each section.
----------------------
logging:
    log_filename: log/harvester.log
    log_level: warning
    log_maxfilesrotation: 30
    log_stdout: false

**Comment by BrandtH22** [comment]
@Hardcore-fs , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
